src/otx/algo/classification/deit_tiny.py

```diff
@@ -78,6 +78,9 @@ class DeitTinyForHLabelCls(ExplainableDeit, MMPretrainHlabelClsModel):
     """DeitTiny Model for hierarchical label classification task."""
 
     def __init__(self, num_classes: int, num_multiclass_heads: int, num_multilabel_classes: int) -> None:
+        self.num_multiclass_heads = num_multiclass_heads
```

Comment:
It seems that all h-label model classes need to save `num_multiclass_heads` and `num_multilabel_classes`, how about passing it to super().__init__() and making it to save them as own attribute?

Reply:
For the `num_multiclass_heads` and `num_multilabel_classes`, it will be removed at the next PR. So, I don't want to enhance the logic at this PR. What do you think?